setup.cfg

```diff
@@ -33,5 +33,8 @@ all_files = 1
 [upload_sphinx]
 upload-dir = docs/_build/html
 
+[flake8]
+max-line-length = 120
```

Comment:
Any reason for changing this the `max-line-length`?

Reply:
I guess there are a LOT of violations. I think it s by default 79, which is a bit on the low side, 120 is quite hight, maybe set it to 100?

Reply:
Yes, The default for flake8 is 79 but for our project it has been set as 90 : https://github.com/apache/incubator-airflow/blob/c7a472ed6b0d8a4720f57ba1140c8cf665757167/tox.ini#L30

Reply:
Ah thanks @kaxil. Lets set it to 90 then! :)

Reply:
thanks for the feedback @Fokko , @kaxil . I update the pr with 90. PTAL.